src/message/connection-endpoint.js

```diff
@@ -57,13 +57,15 @@ var ConnectionEndpoint = function( options, readyCallback ) {
 			// we provide a fake module to patch it if necassary, and look it up
 			// from libs incase of a binary deployment since it's a native module
 			var req = global && global.require ? global.require : require;
+			console.log( process.versions )
```

Comment:
Thanks for noticing. This is actually a debug pr I'm using to print stuff on ci systems due to compilation issues